arm-sql/2014-04-01/swagger/capabilities.json

```diff
@@ -0,0 +1,332 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "Azure SQL Database capabilities",
+        "description": "Describes the Azure SQL capabilities available to a subscription for a given location.",
+        "version": "2014-04-01"
+    },
+    "host": "management.azure.com",
+    "schemes": [
+        "https"
+    ],
+    "consumes": [
+        "application/json"
+    ],
+    "produces": [
+        "application/json"
+    ],
+    "paths": {
+        "/subscriptions/{subscriptionId}/providers/Microsoft.Sql/locations/{locationID}/capabilities": {
+            "get": {
+                "tags": [
+                    "Capabilities"
+                ],
+                "operationId": "Capabilities_Get",
```

Comment:
Isn't this technically Capabilities_List ? Since it returns a list. Or arguably Capabilities_ListByLocation